lib/iex/lib/iex/info.ex

```diff
@@ -290,6 +290,32 @@ defimpl IEx.Info, for: Reference do
   end
 end
 
+defimpl IEx.Info, for: [Date, Time, NaiveDateTime] do
+  sigils = [{Date, "D"}, {Time, "T"}, {NaiveDateTime, "N"}]
+  @sigil sigils[@for]
+  reprs = [{Date, "date"}, {Time, "time"},
```

Comment:
Can we have this as 

```elixir
representations = %{
  Date => "date",
  Time => "time",
  ...
}
```

? It looks easier to read for me, and we don't have to use the "short" form `reprs`. Thoughts?